backend/src/test/java/com/engineering/thesis/backend/controllerIntegrationTests/configration/MockConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package com.engineering.thesis.backend.controllerIntegrationTests.configration;
+
+import com.engineering.thesis.backend.config.jwt.JwtToken;
+import com.engineering.thesis.backend.config.jwt.UnauthorizedHandler;
+import com.engineering.thesis.backend.serviceImpl.user.UserDetailsServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers.springSecurity;
+
+public abstract class MockConfiguration {
```

Comment:
It's important due to necessary mocks. Without them all tests will fail on initialization.